report_server/server.py

```diff
@@ -6,56 +7,59 @@
 from bson import ObjectId
 from google.protobuf.empty_pb2 import Empty
 from google.protobuf.json_format import MessageToDict, ParseDict
-from pymongo import MongoClient
+from pymongo import MongoClient, errors
 
-import employee_report_pb2_grpc, employee_report_pb2
+from protocol_buffers import employee_report_pb2_grpc
 
 
-def convert(report):
+def convert(report, message):
```

Comment:
maybe you should make this function as private or protected method of class?

Reply:
why? PyCharm suggests to make it a function outside of the class

Reply:
But you didn't ask "why?" ?)

Reply:
because PyCharm suggests only now, when function becomes method :-)